packages/fxa-auth-server/test/local/subhub/client.js

```diff
@@ -154,7 +154,7 @@ describe('subhub client', () => {
     it('should throw on unknown user', async () => {
```

Comment:
in the test above ('should list subscriptions for account'), note that subhub for some reason returns a 201, not a 200. Not a big deal, but worth noting.

Reply:
Yeah, that's a subhub issue https://github.com/mozilla/subhub/issues/101#issuecomment-515242286